config/csv_file_defaults.yml

```diff
@@ -1,5 +1,6 @@
 
 Weam:
-  skip_lines_before_header: 0
-  skip_lines_after_header: 1
-  delimiter: ','
+  skipped_first_lines: 0
+  col_sep: ','
+  row_sep: "\r\n"
```

Comment:
Unfortunately, no. Windows uses a different termination than does Unix/OSX. Most cases, the windows termination will be used. However, this is not guaranteed. This option will be presented to the user when uploading. We should make the option understandable for a user though.